README.md

```diff
@@ -1,194 +1,10 @@
-# taproot-workshop
+# taproot-workshop Google Colab notebooks
 
-This repo contains the jupyter notebooks for Optech's [Schnorr/Taproot
-workshops](https://bitcoinops.org/workshops/#taproot-workshop).
+This branch contains the Google Colab version of Optech's 
+[Schnorr/Taproot workshops](https://bitcoinops.org/workshops/#taproot-workshop).
 
-## Introduction
+The notebooks in this branch have been modified slightly to run on Google's
+hosted Colab environment. Expect this branch to be rebased occasionally on
+master.
 
```

Comment:
Does it make sense to link to the Google Colab versions of the notebooks in bullet points below?

Otherwise (as a user) I find myself hunting around for where the notebooks are.

Reply:
I'd say a that the typical user will most likely stay on the master branch when exploring the repository and trying out the Colab notebooks. But it wouldn't cause any harm to add them there.

Your call.

Reply:
I think you’re right. Fine to leave out.